Library/Formula/natalie.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
+  sha256 "943dab1b7ac16555ec3047983a3c397f7ebe7559d3c098c02a6ee6d8e966655e"
+  depends_on :xcode => "6.0"
+
+  def install
+  	bin.install "natalie.swift"
```

Comment:
The code format should be fine now, can you confirm?

Regarding calling `.swift` I also think that's kinda weird, however the project owner keeps his tutorial with the `.swift` suffix. It might get changed on a future release though, but that's up to him and we would need a issue to discuss that better with him. /cc @krzyzanowskim